tests/stubs/python-3.12/pybind11-master/numpy-array-use-type-var/demo/_bindings/aliases/__init__.pyi

```diff
@@ -5,9 +5,9 @@ import typing
 import numpy
 from numpy import random
 
-import demo._bindings.enum
 from demo._bindings.aliases.foreign_method_arg import Bar2 as foreign_type_alias
 from demo._bindings.aliases.foreign_return import get_foo as foreign_class_alias
+from demo._bindings.enum import ConsoleForegroundColor
```

Comment:
Let's not do value imports. They are problematic in a couple of ways.

For example, this would make the following code pass a static analyzer but fail at runtime.

```python 
from demo._bindings.aliases import ConsoleForegroundColor
```